kwanghyun/services/postService.js

```diff
@@ -0,0 +1,30 @@
+const postDao = require("../models/postDao");
+
+const createPosts = async (title, content, userId) => {
+  const createPosts = await postDao.createPost(title, content, userId);
+
+  return createPosts;
```

Comment:
```suggestion
  return postDao.createPost(title, content, userId);
```
위에서 남겨드린 코멘트와 동일한 맥락이며 적용되는 기타 부분들도 일괄적으로 수정될 수 있게 해주세요.